src/components/ShowWinner/ShowWinner.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components/macro'
+
+export default function ShowWinner({ visible, title, players, confetti }) {
+  const winners = players.sort((a, b) => a.score - b.score)
+  const newWinners = []
+
+  return (
+    <WrapperWinner hidden={visible}>
+      <h1>{title}</h1>
+      {determineWinner()}
```

Comment:
It is a bit weird that you call the function in your return statement. And if you filter, you don't need this function anymore, I think.